lib/sidekiq/throttled.rb

```diff
@@ -77,9 +77,8 @@ def setup!
       # @return [Boolean]
       def throttled?(message) # rubocop:disable Metrics/MethodLength
         message = JSON.parse message
-        job = message.fetch("class")   { return false }
-        job = message.fetch("wrapped") { return false } if job == "ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper"
-        jid = message.fetch("jid")     { return false }
+        job = message.fetch("wrapped", message.fetch("class", false))
+        jid = message.fetch("jid", false)
```

Comment:
Let's try `"class"` first, then `"wrapped"`. Also, original logic was short-circuiting and returning `false` if no `"class"` found, which now I see was a bit defective.

```suggestion
        job = message.fetch("class") { message.fetch("wrapped", false) }
        return unless job

        jid = message.fetch("jid", false)
        return unless jid
```

Reply:
With the proposed changes above, you won't need to remove rubocop disable :rofl:

Reply:
afraid you need to do wrapped first, since class will be always "JobWrapper" for rails https://github.com/rails/rails/blob/18707ab17fa492eb25ad2e8f9818a320dc20b823/activejob/lib/active_job/queue_adapters/sidekiq_adapter.rb#L23

Reply:
I fixed the other point of early returning

Reply:
> afraid you need to do wrapped first, since class will be always "JobWrapper" for rails 

Oh. Indeed. Totally overlooked that.